platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/geometry/LatLngBoundsTest.java

```diff
@@ -194,6 +211,66 @@ public void testOuterUnion() {
   }
 
   @Test
+  public void northWest() {
```

Comment:
I'm worried about developers using this class outside offline cases. If wrapping is not tested at the core level either, then it's fine to leave it there but I'd make a note in Javadoc.